tests/test_caching.py

```diff
@@ -41,7 +41,7 @@ def fixture_setup_runner():
 
     yield runner
 
-    runner.stop()
+    runner.cleanup()
```

Comment:
do we still want to stop this ?

Reply:
It probably wouldn't hurt to stop first. The cleanup() method does a force remove. So it'll remove the container even if it is still running. I am not sure it matters much for the test to stop it first.